istioctl/cmd/waypoint.go

```diff
@@ -112,6 +114,9 @@ func waypointCmd() *cobra.Command {
 				FieldManager: "istioctl",
 			})
 			if err != nil {
+				if strings.Contains(err.Error(), missingResourceMessageFragment) {
```

Comment:
nit: you can use `errors.IsNotFound`. I know we do this same check elsewhere though

Reply:
That's much cleaner. I think it's not a nit.